src/plugins/analysis/known_vulnerabilities/docker/scripts/detect_CVE-2021-45608.py

```diff
@@ -19,7 +19,7 @@ def find_kmalloc_calls_in_function(func):
         if call.getName() == '__kmalloc':
             for ref in getReferencesTo(call.getEntryPoint()):  # noqa
                 if func.getBody().contains(ref.getFromAddress()):
-                    print('found __kmalloc call @ {}'.format(ref.getFromAddress()))
```

Comment:
this file is a Ghidra / jython script and cannot be upgraded in this way (which is also the reason the tests fail in the CI)